### DISCUSSION TITLE
Sugestion

### DISCUSSION THREAD

**Discussion by AlyxSharkBite** [mjebrahimi/EasyCompressor#124] [Ideas]
readonly property on the BaseCompressor to return compression method.  ie DeflateCompressor returns "deflate"  

(or returns an enum like CompressionMethod.Deflate) 
This would make it easy to track/log what compression method was used,  

User story:
As a user I want to store a log with a compressed Json object.  I need to track what compression method was used, so I can decompress it correctly if we change the compression used in the project later.   

I have an object like so to track it
long Id (Primary Key)
int UserId
string UserName
DateTimeOffset TimeStamp
string CompressionMethod
byte[] CompressedData

**Comment by mjebrahimi** [comment]
LGTM👍
Can you submit a PR for this one?
